Examples/ObjectiveC/ImageSourceExample.m

```diff
@@ -0,0 +1,37 @@
+
+#import "ImageSourceExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleImageSource = @"ImageSourceExample";
+
+@interface ImageSourceExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ImageSourceExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL: [MGLStyle darkStyleURL]];
+    mapView.delegate = self;
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(43.457, -75.789) zoomLevel:4 animated:NO];
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    MGLCoordinateQuad coordinates = MGLCoordinateQuadMake(CLLocationCoordinate2DMake(46.437, -80.425), CLLocationCoordinate2DMake(37.936, -80.425), CLLocationCoordinate2DMake(37.936, -71.516), CLLocationCoordinate2DMake(46.437, -71.516));
```

Comment:
Hrm, does this wrap well? Might consider doing what you did in the Swift version and manually wrap.